lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -69,11 +78,14 @@ EOS
 sub execute {
     my $self = shift;
     
+    my $input_bam = $self->get_bam();
```

Comment:
I don't know if this matters, but due to having side effects, calling `get_bam()` a second time will crash the program if a model (or build) was specified as an input (even in the PR to this one that I submitted).

The first time its called, it will set `$self->bam_file`, and subsequent times, it will `die` because both `bam_file and model` are set. It doesn't appear to have an effect in the current version of this command as `bam_file` is only called a single time, but it does seem like unexpected behavior from a method called `get_*`.